client/src/base-components/Drawer/Drawer.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { MUI_PLACEMENTS } from '@utils';
+import {
+  Box,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  MuiDrawer,
+} from './Drawer.styled';
+
+export default function Drawer({
+  anchor,
+  children,
+  childrenListStyle,
+  itemsList,
+  itemsListStyle,
+  onClose,
+  onOpen,
+  open,
+  variant,
+  ...props
+}) {
+  return (
+    <MuiDrawer
+      anchor={anchor}
+      childrenListStyle={childrenListStyle}
+      itemsListStyle={itemsListStyle}
+      onClose={onClose}
+      onOpen={onOpen}
+      open={open}
+      variant={variant}
+      {...props}
+    >
+      <List>
+        {itemsList?.map(({ name, icon }, i) => (
+          <ListItem key={i} sx={itemsListStyle}>
+            <ListItemButton>
+              {icon ? <ListItemIcon>{icon}</ListItemIcon> : null}
+              {name ? <ListItemText primary={name} /> : null}
+            </ListItemButton>
+          </ListItem>
+        ))}
+      </List>
+      {children ? <Box sx={childrenListStyle}>{children}</Box> : null}
```

Comment:
if you include children, so you wont add also itemList here 
make sure you checked it here